SA/design_concept/content/poc/architecture.typ

```diff
@@ -1,23 +1,44 @@
 = Architecture <poc-architecture>
-Content: Explain architecture with electron, threepenny, frontend & backend
+This chapter describes the architecture of the PoC application.
 
-== Translation Layer <translation-layer>
+As described in @impl_options_decision, the PoC uses Electron.js and Threepenny-GUI for its implementation.
+This choice results in the applicaiton havig a "remote" user interface, as can be seen in @container-diagram.
```

Comment:
```suggestion
This choice results in the application having a "remote" user interface, as shown in @container-diagram.
```

Reply:
Maybe we could add a citation for the 'remote user interface'. It has been described in this paper: https://www.semanticscholar.org/paper/Client-Server-Architectures-for-Business-Systems-A-Renzel-Keller/587b3772fe5b0365cd94c8d390c7a78524b8797b

(from AppArch lecture slides)